### ISSUE TITLE
1.21.1 - packet mode - not working

### ISSUE THREAD

**Issue by Shadow192** [dadus33/ChatItem#143]
```
[22:22:45 WARN]: java.lang.NullPointerException: Cannot invoke "java.lang.reflect.Method.invoke(Object, Object[])" because "me.dadus33.chatitem.chatmanager.v1.json.JSONManipulator.SAVE_NMS_ITEM_STACK_METHOD" is null
[22:22:45 WARN]:        at ChatItem-2.6.1.jar//me.dadus33.chatitem.chatmanager.v1.json.JSONManipulator.stringifyItemInternal(JSONManipulator.java:279)
[22:22:45 WARN]:        at ChatItem-2.6.1.jar//me.dadus33.chatitem.chatmanager.v1.json.JSONManipulator.stringifyItem(JSONManipulator.java:268)
[22:22:45 WARN]:        at ChatItem-2.6.1.jar//me.dadus33.chatitem.chatmanager.v1.json.JSONManipulator.parse(JSONManipulator.java:77)
[22:22:45 WARN]:        at ChatItem-2.6.1.jar//me.dadus33.chatitem.chatmanager.v1.basecomp.IComponentManager.manageContent(IComponentManager.java:53)
[22:22:45 WARN]:        at ChatItem-2.6.1.jar//me.dadus33.chatitem.chatmanager.v1.listeners.ChatPacketManager.lambda$2(ChatPacketManager.java:169)
[22:22:45 WARN]:        at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
[22:22:45 WARN]:        at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796)
[22:22:45 WARN]:        at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387)
[22:22:45 WARN]:        at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312)
[22:22:45 WARN]:        at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843)
[22:22:45 WARN]:        at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808)
[22:22:45 WARN]:        at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188)
[22:22:45 WARN]: java.lang.reflect.InvocationTargetException
[22:22:45 WARN]:        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:118)
[22:22:45 WARN]:        at java.base/java.lang.reflect.Method.invoke(Method.java:580)
[22:22:45 WARN]:        at ChatItem-2.6.1.jar//me.dadus33.chatitem.platform.hook.PaperPlatform.jsonToBaseComponent(PaperPlatform.java:126)
[22:22:45 WARN]:        at ChatItem-2.6.1.jar//me.dadus33.chatitem.chatmanager.v1.PacketEditingChatManager.createSystemChatPacket(PacketEditingChatManager.java:83)
[22:22:45 WARN]:        at ChatItem-2.6.1.jar//me.dadus33.chatitem.chatmanager.v1.basecomp.hook.ComponentNMSManager.writeJson(ComponentNMSManager.java:41)
[22:22:45 WARN]:        at ChatItem-2.6.1.jar//me.dadus33.chatitem.chatmanager.v1.basecomp.IComponentManager.manageContent(IComponentManager.java:56)
[22:22:45 WARN]:        at ChatItem-2.6.1.jar//me.dadus33.chatitem.chatmanager.v1.listeners.ChatPacketManager.lambda$2(ChatPacketManager.java:169)
[22:22:45 WARN]:        at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
[22:22:45 WARN]:        at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796)
[22:22:45 WARN]:        at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387)
[22:22:45 WARN]:        at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312)
[22:22:45 WARN]:        at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843)
[22:22:45 WARN]:        at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808)
[22:22:45 WARN]:        at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188)
[22:22:45 WARN]: Caused by: com.google.gson.JsonParseException: Failed to parse either. First: No key contents in MapLike[{"action":"show_item","value":"{}"}]; Second: No key id in MapLike[{}]
[22:22:45 WARN]:        at com.mojang.serialization.DataResult$Error.getOrThrow(DataResult.java:287)
[22:22:45 WARN]:        at net.minecraft.network.chat.Component$Serializer.deserialize(Component.java:296)
[22:22:45 WARN]:        at net.minecraft.network.chat.Component$Serializer.fromJsonLenient(Component.java:326)
[22:22:45 WARN]:        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
[22:22:45 WARN]:        ... 13 more
[22:22:45 WARN]: [ChatItem] Can't create a new packet for json {"text":"","extra":[{"text":" ┃","italic":false,"color":"dark_gray"},{"text":" Chairman ","italic":false,"color":"black","bold":true},{"text":"┃","italic":false,"color":"dark_gray","bold":false},{"text":" ","italic":false},{"text":"_ShadowXD","italic":false,"color":"black"},{"text":"  ","italic":false},{"text":"› ","italic":false,"color":"dark_gray"},{"extra":[{"bold":true,"italic":true,"color":"aqua","text":"Diamond Axe x1"},{"reset":true,"color":"white","text":""}],"text":"","hoverEvent":{"action":"show_item","value":"{}"}}]}
```

The above error is returning when doing [i] on a 1.21.1 purpur server (latest build) on packet as chat manager mode.

**Comment by Elikill58** [comment]
This is not latest build. You can find latest dev build here: https://github.com/dadus33-plugins/ChatItem/actions/runs/10770218605 or on discord serveur

**Comment by Shadow192** [comment]
Fixed in latest snapshop (2.6.2)
